.github/workflows/run_tests.yaml

```diff
@@ -4,6 +4,8 @@ on:
     branches:
       - main
   pull_request:
+  schedule:
+    - cron: '23 2 * * *'
```

Comment:
correct

Reply:
Any particular meaning to that time? Not that I'm opposed

Reply:
nope, just avoiding top of the hour to avoid high github loads as recommended: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule

happy to change to something else if we want to hide an easter egg here :)

Reply:
223 is prime so that's "easter egg" enough for me